golden/clr-angular.d.ts

```diff
@@ -349,9 +349,11 @@ export declare class ClrDatagridFilter<T = any> extends DatagridFilterRegistrar<
     toggle(): void;
 }
 
-export interface ClrDatagridFilterInterface<T> {
+export interface ClrDatagridFilterInterface<T, S = any> {
     changes: Observable<any>;
+    state?: S;
     accepts(item: T): boolean;
+    equals?(other: ClrDatagridFilterInterface<any, any>): boolean;
```

Comment:
Thinking about it, I think we can get away with `ClrDatagridFilterInterface<T, any>`. The datagrid items are the same for all filters in that datagrid.

Reply:
Done